test/unit/org/apache/cassandra/db/guardrails/GuardrailsTest.java

```diff
@@ -122,7 +122,7 @@ public void testThresholdUsers() throws Throwable
         assertValid(() -> guard.guard(100, "y", superClientState));
 
         // value over fail threshold
-        assertFails(() -> guard.guard(101, "z", null), "Aborting: for z, 101 > 100");
+        assertWarns(() -> guard.guard(101, "z", null), "Aborting: for z, 101 > 100");
```

Comment:
Add a note here that we're warning instead of failing here to prevent blowing up on processes not client facing. Otherwise this is confusing imo.

Reply:
I was referring to the test itself. On the comment in the previous line i.e.

Reply:
I see, I have added that comment.